www/calendar/app-calendar.less

```diff
@@ -155,6 +155,10 @@
         }
         .tui-full-calendar-popup {
             border-radius: @variables_radius_L;
+            @media screen and (max-width : @browser_media-medium-screen){
+                position: fixed;
+                top: 5.5rem;
```

Comment:
Using `positon: fixed` without a max-height is risky:
![image](https://github.com/user-attachments/assets/4f2051d0-e0f6-4eb4-98f3-bcd21b64a0b8)
In this example, I can't reach the "Save" button because it is hidden outside of the screen and since the position is fixed, scrolling doesn't change anything.
A fix could be to add a max-height to the modal and make it scrollable with the "overflow" property.